browser-test/src/trusted_intermediary.test.ts

```diff
@@ -656,6 +656,12 @@ test.describe('Trusted intermediaries', () => {
     await adminTiGroups.expectGroupExist('group name', 'group description')
     await validateScreenshot(page, 'ti-groups-page')
 
+    // validate error message if empty name
+    await adminTiGroups.editGroup('group name')
+    await page.click('text="Add"')
+    await validateScreenshot(page, 'manage-ti-group-members-empty-email')
```

Comment:
(blocking) I don't think we need to add a screen. Instead can you assert that the toast has the expected text.